AEPServices/Sources/network/DefaultHeadersFormatter.swift

```diff
@@ -0,0 +1,62 @@
+/*
+ Copyright 2021 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+
+///
+/// Helper class used to format the default http headers
+///
+public struct DefaultHeadersFormatter {
+
+    typealias httpHeaderConstants = HttpConnectionConstants.Header
+
+    ///
+    /// Gets the formatted locale
+    /// - Parameter unformattedLocale: The unformatted locale as a string
+    /// - Returns: The formatted locale as a string or nil if not able to format it.
+    ///
+    public static func getFormattedLocale(unformattedLocale: String) -> String {
```

Comment:
`_ unformattedLocale`

Reply:
Decided to change the name as well. Lmk what you think.